tests/runtests.pl

```diff
@@ -1142,24 +1142,40 @@ sub singletest_shouldrun {
         if(!$info_keywords[0]) {
             $why = "missing the <keywords> section!";
         }
+        # Only evaluate keywords if the section is present.
+        else {
+            my @info_features = getpart("client", "features");
+            # Prefix features with "feat:" and add to keywords list.
+            push @info_keywords, map { "feat:" . lc($_) } @info_features;
```

Comment:
You could just eliminate the @info_features variable altogether and just call getpart here directly.